--------------------Moonraker Log Start--------------------
data_path: /home/pi/printer_data
is_default_data_path: False
config_file: /home/pi/klipper_config/moonraker.conf
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
log_file: /home/pi/klipper_logs/moonraker.log
software_version: v0.7.1-773-g362bc1a
python_version: 3.7.3 (default, Jan 22 2021, 20:04:44)  [GCC 8.3.0]

#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
max_upload_size = 1024

[file_manager]
enable_object_processing = True

[authorization]
cors_domains = 
	https://my.mainsail.xyz
	http://my.mainsail.xyz
	http://*.local
	http://*.lan
trusted_clients = 
	10.0.0.0/8
	127.0.0.0/8
	169.254.0.0/16
	172.16.0.0/12
	192.168.0.0/16
	FE80::/10
	::1/128

[octoprint_compat]

[history]

[announcements]
subscriptions = 
	mainsail

[update_manager]
refresh_interval = 168
enable_auto_refresh = True

[update_manager mainsail]
type = web
channel = stable
repo = mainsail-crew/mainsail
path = ~/mainsail

[update_manager crowsnest]
type = git_repo
path = ~/crowsnest
origin = https://github.com/mainsail-crew/crowsnest.git
managed_services = crowsnest

[update_manager sonar]
type = git_repo
path = ~/sonar
origin = https://github.com/mainsail-crew/sonar.git
primary_branch = main
managed_services = sonar

[power mainpower]
type = gpio
pin = gpiochip0/gpio18
off_when_shutdown = true
restart_klipper_when_powered = true
initial_state = off
locked_while_printing = false
on_when_job_queued = false

#################################################################
All Configuration Files:
/home/pi/klipper_config/moonraker.conf
#################################################################
Unsafe Shutdown Count: 59

System Info:

***python***
  version: sys.version_info(major=3, minor=7, micro=3, releaselevel='final', serial=0)
  version_string: 3.7.3 (default, Jan 22 2021, 20:04:44)  [GCC 8.3.0]

***cpu_info***
  cpu_count: 4
  bits: 32bit
  processor: armv7l
  cpu_desc: ARMv7 Processor rev 3 (v7l)
  serial_number: 100000009514db42
  hardware_desc: BCM2711
  model: Raspberry Pi 4 Model B Rev 1.4
  total_memory: 7904580
  memory_units: kB

***sd_info***

***distribution***
  name: Raspbian GNU/Linux 10 (buster)
  id: raspbian
  version: 10
  version_parts: {'major': '10', 'minor': '', 'build_number': ''}
  like: debian
  codename: buster
  release_info: {'name': 'MainsailOS', 'version_id': '0.7.1', 'codename': 'buster', 'id': 'mainsailos'}

***virtualization***
  virt_type: none
  virt_identifier: none

***network***

***canbus***
libgpiod version: 1.2

Systemd unit moonraker.service:
unit_name: moonraker.service
is_default: True
Properties:
**Requires=['system.slice', 'network-online.target', 'sysinit.target', '-.mount']
**After=['systemd-journald.socket', 'network-online.target', '-.mount', 'sysinit.target', 'basic.target', 'system.slice', 'klipper.service']
**SupplementaryGroups=['moonraker-admin']
**EnvironmentFiles=/home/pi/printer_data/systemd/moonraker.env
**ExecStart=/home/pi/moonraker-env/bin/python $MOONRAKER_ARGS
**WorkingDirectory=/home/pi/moonraker
**FragmentPath=/etc/systemd/system/moonraker.service
**Description=API Server for Klipper SV1
**User=pi

Systemd unit klipper.service:
unit_name: klipper.service
is_default: True
Properties:
**Description=Starts Klipper and provides klippy Unix Domain Socket API
**ExecStart=/home/pi/klippy-env/bin/python /home/pi/klipper/klippy/klippy.py ${KLIPPER_CONFIG} -l ${KLIPPER_LOG} -a ${KLIPPER_SOCKET}
**FragmentPath=/etc/systemd/system/klipper.service
Klipper Version: v0.10.0-635-gcfb633a0
2022-12-06 03:57:41,904 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.1.66
2022-12-06 03:57:41,906 [app.py:log_request()] - 101 GET /websocket (192.168.1.66) [_TRUSTED_USER_] 3.76ms
2022-12-06 03:57:41,906 [websockets.py:open()] - Websocket Opened: ID: 2818741456, Proxied: True, User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36, Host Name: 192.168.1.90
2022-12-06 03:57:41,914 [websockets.py:_handle_identify()] - Websocket 2818741456 Client Identified - Name: mainsail, Version: 2.4.0, Type: web
2022-12-06 04:00:04,677 [app.py:log_request()] - 201 POST /server/files/upload (192.168.1.66) [_TRUSTED_USER_] 10.79ms
2022-12-06 04:00:04,745 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2022-12-06 04:00:06,002 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2022-12-06 04:00:06,370 [power.py:_schedule_firmware_restart()] - Power Device mainpower: Sending FIRMWARE_RESTART command to Klippy
2022-12-06 04:00:06,371 [klippy_connection.py:_init_klippy_connection()] - Webhooks Subscribed
2022-12-06 04:00:06,376 [klippy_connection.py:_init_klippy_connection()] - GCode Output Subscribed
2022-12-06 04:00:06,380 [klippy_connection.py:_check_ready()] - 
pin PA5 used multiple times in config

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.
Printer is halted

2022-12-06 04:00:07,990 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2022-12-06 04:00:09,247 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2022-12-06 04:00:09,546 [klippy_connection.py:_init_klippy_connection()] - Webhooks Subscribed
2022-12-06 04:00:09,548 [klippy_connection.py:_init_klippy_connection()] - GCode Output Subscribed
2022-12-06 04:00:09,552 [klippy_connection.py:_check_ready()] - 
pin PA5 used multiple times in config

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.
Printer is halted

2022-12-06 04:07:07,156 [app.py:log_request()] - 201 POST /server/files/upload (192.168.1.66) [_TRUSTED_USER_] 20.01ms
2022-12-06 04:08:09,833 [app.py:log_request()] - 201 POST /server/files/upload (192.168.1.66) [_TRUSTED_USER_] 17.48ms
2022-12-06 04:08:10,476 [klippy_connection.py:close()] - Error closing Klippy Unix Socket
Traceback (most recent call last):
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 577, in close
    await self.writer.wait_closed()
  File "/usr/lib/python3.7/asyncio/streams.py", line 323, in wait_closed
    await self._protocol._closed
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 143, in _read_stream
    data = await reader.readuntil(b'\x03')
  File "/usr/lib/python3.7/asyncio/streams.py", line 588, in readuntil
    await self._wait_for_data('readuntil')
  File "/usr/lib/python3.7/asyncio/streams.py", line 473, in _wait_for_data
    await self._waiter
  File "/usr/lib/python3.7/asyncio/selector_events.py", line 801, in _read_ready__data_received
    data = self._sock.recv(self.max_size)
ConnectionResetError: [Errno 104] Connection reset by peer
2022-12-06 04:08:10,476 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2022-12-06 04:08:10,479 [websockets.py:build_error()] - JSON-RPC Request Error: 503
Klippy Disconnected
Traceback (most recent call last):
  File "/home/pi/moonraker/moonraker/websockets.py", line 253, in execute_method
    result = await callback(params)
  File "/home/pi/moonraker/moonraker/websockets.py", line 351, in func
    user=sc.user_info))
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 481, in request
    return await self._request_standard(web_request)
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 536, in _request_standard
    return await base_request.wait()
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 608, in wait
    raise self.response
utils.ServerError: Klippy Disconnected
2022-12-06 04:08:11,737 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2022-12-06 04:08:12,035 [klippy_connection.py:_init_klippy_connection()] - Webhooks Subscribed
2022-12-06 04:08:12,036 [klippy_connection.py:_init_klippy_connection()] - GCode Output Subscribed
2022-12-06 04:08:12,040 [klippy_connection.py:_check_ready()] - 
Internal error during connect: 'PrinterMultiPin' object has no attribute 'setup_adc_callback'

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.
Printer is halted

2022-12-06 04:08:35,408 [app.py:log_request()] - 201 POST /server/files/upload (192.168.1.66) [_TRUSTED_USER_] 15.39ms
2022-12-06 04:08:49,322 [app.py:log_request()] - 201 POST /server/files/upload (192.168.1.66) [_TRUSTED_USER_] 20.37ms
2022-12-06 04:08:49,965 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2022-12-06 04:08:51,224 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2022-12-06 04:08:51,524 [klippy_connection.py:_init_klippy_connection()] - Webhooks Subscribed
2022-12-06 04:08:51,526 [klippy_connection.py:_init_klippy_connection()] - GCode Output Subscribed
2022-12-06 04:08:51,530 [klippy_connection.py:_check_ready()] - 
Unknown pin chip name 'multi_pin'

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.
Printer is halted

2022-12-06 04:11:35,310 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2022-12-06 04:11:36,573 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2022-12-06 04:11:36,876 [klippy_connection.py:_init_klippy_connection()] - Webhooks Subscribed
2022-12-06 04:11:36,878 [klippy_connection.py:_init_klippy_connection()] - GCode Output Subscribed
2022-12-06 04:11:36,883 [klippy_connection.py:_check_ready()] - 
Unknown pin chip name 'multi_pin'

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.
Printer is halted

2022-12-06 04:11:55,313 [app.py:log_request()] - 201 POST /server/files/upload (192.168.1.66) [_TRUSTED_USER_] 18.48ms
2022-12-06 04:12:06,703 [app.py:log_request()] - 201 POST /server/files/upload (192.168.1.66) [_TRUSTED_USER_] 17.79ms
2022-12-06 04:12:16,172 [app.py:log_request()] - 201 POST /server/files/upload (192.168.1.66) [_TRUSTED_USER_] 17.94ms
2022-12-06 04:12:23,198 [app.py:log_request()] - 201 POST /server/files/upload (192.168.1.66) [_TRUSTED_USER_] 19.02ms
2022-12-06 04:12:23,841 [klippy_connection.py:close()] - Error closing Klippy Unix Socket
Traceback (most recent call last):
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 577, in close
    await self.writer.wait_closed()
  File "/usr/lib/python3.7/asyncio/streams.py", line 323, in wait_closed
    await self._protocol._closed
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 143, in _read_stream
    data = await reader.readuntil(b'\x03')
  File "/usr/lib/python3.7/asyncio/streams.py", line 588, in readuntil
    await self._wait_for_data('readuntil')
  File "/usr/lib/python3.7/asyncio/streams.py", line 473, in _wait_for_data
    await self._waiter
  File "/usr/lib/python3.7/asyncio/selector_events.py", line 801, in _read_ready__data_received
    data = self._sock.recv(self.max_size)
ConnectionResetError: [Errno 104] Connection reset by peer
2022-12-06 04:12:23,842 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2022-12-06 04:12:23,845 [websockets.py:build_error()] - JSON-RPC Request Error: 503
Klippy Disconnected
Traceback (most recent call last):
  File "/home/pi/moonraker/moonraker/websockets.py", line 253, in execute_method
    result = await callback(params)
  File "/home/pi/moonraker/moonraker/websockets.py", line 351, in func
    user=sc.user_info))
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 481, in request
    return await self._request_standard(web_request)
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 536, in _request_standard
    return await base_request.wait()
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 608, in wait
    raise self.response
utils.ServerError: Klippy Disconnected
2022-12-06 04:12:25,102 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2022-12-06 04:12:25,403 [klippy_connection.py:_init_klippy_connection()] - Webhooks Subscribed
2022-12-06 04:12:25,405 [klippy_connection.py:_init_klippy_connection()] - GCode Output Subscribed
2022-12-06 04:12:25,409 [klippy_connection.py:_check_ready()] - 
Internal error during connect: 'PrinterMultiPin' object has no attribute 'setup_adc_callback'

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.
Printer is halted

2022-12-06 04:13:40,664 [app.py:log_request()] - 201 POST /server/files/upload (192.168.1.66) [_TRUSTED_USER_] 17.54ms
2022-12-06 04:13:45,982 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2022-12-06 04:13:47,242 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2022-12-06 04:13:47,539 [klippy_connection.py:_init_klippy_connection()] - Webhooks Subscribed
2022-12-06 04:13:47,540 [klippy_connection.py:_init_klippy_connection()] - GCode Output Subscribed
2022-12-06 04:13:47,544 [klippy_connection.py:_check_ready()] - 
Internal error during connect: 'PrinterMultiPin' object has no attribute 'setup_adc_callback'

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.
Printer is halted

2022-12-06 04:14:49,453 [app.py:log_request()] - 201 POST /server/files/upload (192.168.1.66) [_TRUSTED_USER_] 18.90ms
2022-12-06 04:14:50,095 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2022-12-06 04:14:51,360 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2022-12-06 04:14:51,662 [klippy_connection.py:_init_klippy_connection()] - Webhooks Subscribed
2022-12-06 04:14:51,664 [klippy_connection.py:_init_klippy_connection()] - GCode Output Subscribed
2022-12-06 04:14:51,669 [klippy_connection.py:_check_ready()] - 
pin PA5 used multiple times in config

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.
Printer is halted

2022-12-06 04:15:07,280 [app.py:log_request()] - 201 POST /server/files/upload (192.168.1.66) [_TRUSTED_USER_] 19.32ms
2022-12-06 04:15:07,924 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2022-12-06 04:15:09,185 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2022-12-06 04:15:09,514 [klippy_connection.py:_init_klippy_connection()] - Webhooks Subscribed
2022-12-06 04:15:09,520 [klippy_connection.py:_init_klippy_connection()] - GCode Output Subscribed
2022-12-06 04:15:12,522 [job_state.py:_handle_started()] - Job state initialized: standby
2022-12-06 04:15:12,795 [klippy_connection.py:_check_ready()] - Klippy ready
2022-12-06 04:15:13,286 [data_store.py:_init_sensors()] - Configuring available sensors: ['temperature_sensor raspberry_pi', 'temperature_sensor SKR_temp', 'temperature_sensor thermexp', 'temperature_sensor toolhead_MCU', 'temperature_sensor chamber', 'temperature_fan Stepper_Left', 'temperature_fan Stepper_Righ', 'temperature_fan electronics_bay', 'heater_bed', 'extruder']
2022-12-06 04:17:52,317 [app.py:log_request()] - 201 POST /server/files/upload (192.168.1.66) [_TRUSTED_USER_] 15.33ms
2022-12-06 04:17:53,430 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2022-12-06 04:17:54,689 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2022-12-06 04:17:55,024 [klippy_connection.py:_init_klippy_connection()] - Webhooks Subscribed
2022-12-06 04:17:55,026 [klippy_connection.py:_init_klippy_connection()] - GCode Output Subscribed
2022-12-06 04:17:55,030 [klippy_connection.py:_check_ready()] - 
Option 'temperature1' in section 'thermistor SL' must be specified

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.
Printer is halted

2022-12-06 04:19:30,928 [app.py:log_request()] - 201 POST /server/files/upload (192.168.1.66) [_TRUSTED_USER_] 17.43ms
2022-12-06 04:19:31,586 [klippy_connection.py:close()] - Error closing Klippy Unix Socket
Traceback (most recent call last):
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 577, in close
    await self.writer.wait_closed()
  File "/usr/lib/python3.7/asyncio/streams.py", line 323, in wait_closed
    await self._protocol._closed
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 143, in _read_stream
    data = await reader.readuntil(b'\x03')
  File "/usr/lib/python3.7/asyncio/streams.py", line 588, in readuntil
    await self._wait_for_data('readuntil')
  File "/usr/lib/python3.7/asyncio/streams.py", line 473, in _wait_for_data
    await self._waiter
  File "/usr/lib/python3.7/asyncio/selector_events.py", line 801, in _read_ready__data_received
    data = self._sock.recv(self.max_size)
ConnectionResetError: [Errno 104] Connection reset by peer
2022-12-06 04:19:31,586 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2022-12-06 04:19:31,590 [websockets.py:build_error()] - JSON-RPC Request Error: 503
Klippy Disconnected
Traceback (most recent call last):
  File "/home/pi/moonraker/moonraker/websockets.py", line 253, in execute_method
    result = await callback(params)
  File "/home/pi/moonraker/moonraker/websockets.py", line 351, in func
    user=sc.user_info))
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 481, in request
    return await self._request_standard(web_request)
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 536, in _request_standard
    return await base_request.wait()
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 608, in wait
    raise self.response
utils.ServerError: Klippy Disconnected
2022-12-06 04:19:32,847 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2022-12-06 04:19:33,146 [klippy_connection.py:_init_klippy_connection()] - Webhooks Subscribed
2022-12-06 04:19:33,151 [klippy_connection.py:_init_klippy_connection()] - GCode Output Subscribed
2022-12-06 04:19:35,682 [klippy_connection.py:_check_ready()] - 
Pin 'SL' is not a valid pin name on mcu 'mcu'

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.
Printer is halted

2022-12-06 04:19:51,161 [app.py:log_request()] - 201 POST /server/files/upload (192.168.1.66) [_TRUSTED_USER_] 18.03ms
2022-12-06 04:19:51,261 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2022-12-06 04:19:52,520 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2022-12-06 04:19:52,821 [klippy_connection.py:_init_klippy_connection()] - Webhooks Subscribed
2022-12-06 04:19:52,823 [klippy_connection.py:_init_klippy_connection()] - GCode Output Subscribed
2022-12-06 04:19:52,828 [klippy_connection.py:_check_ready()] - 
Unknown pin chip name 'temperature_sensor'

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.
Printer is halted

2022-12-06 04:20:09,322 [app.py:log_request()] - 201 POST /server/files/upload (192.168.1.66) [_TRUSTED_USER_] 19.59ms
2022-12-06 04:20:09,968 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2022-12-06 04:20:11,227 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2022-12-06 04:20:11,540 [klippy_connection.py:_init_klippy_connection()] - Webhooks Subscribed
2022-12-06 04:20:11,545 [klippy_connection.py:_init_klippy_connection()] - GCode Output Subscribed
2022-12-06 04:20:14,547 [job_state.py:_handle_started()] - Job state initialized: standby
2022-12-06 04:20:14,822 [klippy_connection.py:_check_ready()] - Klippy ready
2022-12-06 04:20:15,314 [data_store.py:_init_sensors()] - Configuring available sensors: ['temperature_sensor raspberry_pi', 'temperature_sensor SKR_temp', 'temperature_sensor thermexp', 'temperature_sensor toolhead_MCU', 'temperature_sensor chamber', 'temperature_fan Stepper_Left', 'temperature_fan Stepper_Righ', 'temperature_fan electronics_bay', 'heater_bed', 'extruder']
2022-12-06 04:23:29,515 [app.py:log_request()] - 201 POST /server/files/upload (192.168.1.66) [_TRUSTED_USER_] 18.82ms
2022-12-06 04:23:32,949 [app.py:log_request()] - 201 POST /server/files/upload (192.168.1.66) [_TRUSTED_USER_] 13.90ms
2022-12-06 04:23:34,062 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2022-12-06 04:23:35,320 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2022-12-06 04:23:35,655 [klippy_connection.py:_init_klippy_connection()] - Webhooks Subscribed
2022-12-06 04:23:35,661 [klippy_connection.py:_init_klippy_connection()] - GCode Output Subscribed
2022-12-06 04:23:38,665 [job_state.py:_handle_started()] - Job state initialized: standby
2022-12-06 04:23:38,940 [klippy_connection.py:_check_ready()] - Klippy ready
2022-12-06 04:23:39,432 [data_store.py:_init_sensors()] - Configuring available sensors: ['temperature_sensor raspberry_pi', 'temperature_sensor SKR_temp', 'temperature_sensor thermexp', 'temperature_sensor toolhead_MCU', 'temperature_sensor chamber', 'temperature_fan Stepper_Left', 'temperature_fan Stepper_Righ', 'temperature_fan electronics_bay', 'heater_bed', 'extruder']
2022-12-06 04:27:13,647 [app.py:log_request()] - 404 GET /server/files/gcodes/.thumbs/tapeholder-32x32.png?timestamp=1670103302.5494552 (192.168.1.66) [No User] 17.04ms
2022-12-06 04:27:36,394 [app.py:log_request()] - 404 GET /server/files/gcodes/.thumbs/tapeholder-32x32.png?timestamp=1670103302.5494552 (192.168.1.66) [No User] 6.78ms
2022-12-06 04:29:56,250 [websockets.py:on_close()] - Websocket Closed: ID: 2818741456 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 4.34
2022-12-06 04:29:58,738 [klippy_connection.py:_process_status_update()] - Klippy has shutdown
2022-12-06 04:29:58,738 [power.py:_power_off_on_shutdown()] - Powering off device 'mainpower' due to klippy shutdown
2022-12-06 04:29:58,741 [proc_stats.py:_handle_shutdown()] - 
Moonraker System Usage Statistics:
System Time: 1670322569.107021, Usage: 2.09%, Memory: 29524 kB
System Time: 1670322570.107952, Usage: 2.06%, Memory: 29524 kB
System Time: 1670322571.109497, Usage: 2.14%, Memory: 29524 kB
System Time: 1670322572.110188, Usage: 2.38%, Memory: 29524 kB
System Time: 1670322573.111829, Usage: 2.01%, Memory: 29524 kB
System Time: 1670322574.113457, Usage: 2.1%, Memory: 29524 kB
System Time: 1670322575.115915, Usage: 5.94%, Memory: 29524 kB
System Time: 1670322576.115743, Usage: 2.38%, Memory: 29524 kB
System Time: 1670322577.117280, Usage: 2.16%, Memory: 29524 kB
System Time: 1670322578.118338, Usage: 2.11%, Memory: 29524 kB
System Time: 1670322579.119919, Usage: 2.19%, Memory: 29524 kB
System Time: 1670322580.121983, Usage: 2.1%, Memory: 29524 kB
System Time: 1670322581.124517, Usage: 2.12%, Memory: 29524 kB
System Time: 1670322582.125669, Usage: 2.44%, Memory: 29524 kB
System Time: 1670322583.126825, Usage: 2.15%, Memory: 29524 kB
System Time: 1670322584.128132, Usage: 2.07%, Memory: 29524 kB
System Time: 1670322585.131084, Usage: 6.35%, Memory: 29524 kB
System Time: 1670322586.130740, Usage: 2.2%, Memory: 29524 kB
System Time: 1670322587.131329, Usage: 2.12%, Memory: 29524 kB
System Time: 1670322588.132332, Usage: 2.1%, Memory: 29524 kB
System Time: 1670322589.133737, Usage: 2.14%, Memory: 29524 kB
System Time: 1670322590.134507, Usage: 2.06%, Memory: 29524 kB
System Time: 1670322591.135124, Usage: 2.11%, Memory: 29524 kB
System Time: 1670322592.135887, Usage: 2.46%, Memory: 29524 kB
System Time: 1670322593.137371, Usage: 2.1%, Memory: 29524 kB
System Time: 1670322594.137469, Usage: 2.36%, Memory: 29524 kB
System Time: 1670322595.141716, Usage: 6.36%, Memory: 29524 kB
System Time: 1670322596.136832, Usage: 1.75%, Memory: 29524 kB
System Time: 1670322597.137667, Usage: 1.14%, Memory: 29524 kB
System Time: 1670322598.139125, Usage: 0.75%, Memory: 29524 kB
CPU Temperature: 35.537
2022-12-06 04:29:58,755 [proc_stats.py:_handle_shutdown()] - Throttled Flags: 
2022-12-06 05:26:21,698 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.1.66
2022-12-06 14:19:36,272 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.1.181
2022-12-06 14:19:36,273 [app.py:log_request()] - 101 GET /websocket (192.168.1.181) [_TRUSTED_USER_] 3.85ms
2022-12-06 14:19:36,274 [websockets.py:open()] - Websocket Opened: ID: 2788503344, Proxied: True, User Agent: Mozilla/5.0 (Linux; Android 12; SM-F711U1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Mobile Safari/537.36, Host Name: 192.168.1.90
2022-12-06 14:19:36,294 [websockets.py:_handle_identify()] - Websocket 2788503344 Client Identified - Name: mainsail, Version: 2.4.0, Type: web
2022-12-06 14:19:39,244 [power.py:_schedule_firmware_restart()] - Power Device mainpower: Sending FIRMWARE_RESTART command to Klippy
2022-12-06 14:19:40,298 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2022-12-06 14:19:41,555 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2022-12-06 14:19:41,865 [klippy_connection.py:_init_klippy_connection()] - Webhooks Subscribed
2022-12-06 14:19:41,870 [klippy_connection.py:_init_klippy_connection()] - GCode Output Subscribed
2022-12-06 14:19:44,872 [job_state.py:_handle_started()] - Job state initialized: standby
2022-12-06 14:19:45,145 [klippy_connection.py:_check_ready()] - Klippy ready
2022-12-06 14:19:45,626 [data_store.py:_init_sensors()] - Configuring available sensors: ['temperature_sensor raspberry_pi', 'temperature_sensor SKR_temp', 'temperature_sensor thermexp', 'temperature_sensor toolhead_MCU', 'temperature_sensor chamber', 'temperature_fan Stepper_Left', 'temperature_fan Stepper_Righ', 'temperature_fan electronics_bay', 'heater_bed', 'extruder']
2022-12-06 14:20:11,695 [klippy_connection.py:_process_status_update()] - Klippy has shutdown
2022-12-06 14:20:11,696 [power.py:_power_off_on_shutdown()] - Powering off device 'mainpower' due to klippy shutdown
2022-12-06 14:20:11,702 [proc_stats.py:_handle_shutdown()] - 
Moonraker System Usage Statistics:
System Time: 1670357981.721109, Usage: 5.3%, Memory: 29564 kB
System Time: 1670357982.721629, Usage: 1.74%, Memory: 29564 kB
System Time: 1670357983.725454, Usage: 1.13%, Memory: 29564 kB
System Time: 1670357984.727458, Usage: 2.32%, Memory: 29564 kB
System Time: 1670357985.724095, Usage: 6.1%, Memory: 29564 kB
System Time: 1670357986.727715, Usage: 3.99%, Memory: 29788 kB
System Time: 1670357987.729175, Usage: 1.94%, Memory: 29788 kB
System Time: 1670357988.730880, Usage: 2.05%, Memory: 29788 kB
System Time: 1670357989.731776, Usage: 5.79%, Memory: 29788 kB
System Time: 1670357990.732632, Usage: 2.9%, Memory: 29788 kB
System Time: 1670357991.732730, Usage: 2.05%, Memory: 29788 kB
System Time: 1670357992.734580, Usage: 2.09%, Memory: 29788 kB
System Time: 1670357993.734461, Usage: 2.64%, Memory: 29788 kB
System Time: 1670357994.736266, Usage: 2.56%, Memory: 29788 kB
System Time: 1670357995.737249, Usage: 2.07%, Memory: 29788 kB
System Time: 1670357996.739311, Usage: 2.96%, Memory: 29788 kB
System Time: 1670357997.741697, Usage: 2.12%, Memory: 29788 kB
System Time: 1670357998.743623, Usage: 2.16%, Memory: 29788 kB
System Time: 1670357999.747462, Usage: 5.79%, Memory: 29788 kB
System Time: 1670358000.747897, Usage: 2.31%, Memory: 29788 kB
System Time: 1670358001.748768, Usage: 2.14%, Memory: 29788 kB
System Time: 1670358002.749079, Usage: 2.17%, Memory: 29788 kB
System Time: 1670358003.749979, Usage: 2.09%, Memory: 29788 kB
System Time: 1670358004.750998, Usage: 2.09%, Memory: 29788 kB
System Time: 1670358005.751188, Usage: 2.25%, Memory: 29788 kB
System Time: 1670358006.753646, Usage: 2.4%, Memory: 29788 kB
System Time: 1670358007.749789, Usage: 2.33%, Memory: 29788 kB
System Time: 1670358008.750623, Usage: 1.04%, Memory: 29788 kB
System Time: 1670358009.752184, Usage: 4.26%, Memory: 29788 kB
System Time: 1670358010.752203, Usage: 1.63%, Memory: 29788 kB
CPU Temperature: 37.485
2022-12-06 14:20:11,718 [proc_stats.py:_handle_shutdown()] - Throttled Flags: 
2022-12-06 14:20:46,289 [websockets.py:write_to_socket()] - Websocket closed while writing: 2788503344
2022-12-06 14:20:51,278 [websockets.py:on_close()] - Websocket Closed: ID: 2788503344 Close Code: None, Close Reason: None, Pong Time Elapsed: 44.99
2022-12-06 14:29:35,415 [app.py:log_request()] - 101 GET /websocket (192.168.1.181) [_TRUSTED_USER_] 2.87ms
2022-12-06 14:29:35,415 [websockets.py:open()] - Websocket Opened: ID: 2778639216, Proxied: True, User Agent: Mozilla/5.0 (Linux; Android 12; SM-F711U1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Mobile Safari/537.36, Host Name: 192.168.1.90
2022-12-06 14:29:35,452 [websockets.py:_handle_identify()] - Websocket 2778639216 Client Identified - Name: mainsail, Version: 2.4.0, Type: web
2022-12-06 14:29:39,850 [power.py:_schedule_firmware_restart()] - Power Device mainpower: Sending FIRMWARE_RESTART command to Klippy
2022-12-06 14:29:40,907 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2022-12-06 14:29:42,164 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2022-12-06 14:29:42,464 [klippy_connection.py:_init_klippy_connection()] - Webhooks Subscribed
2022-12-06 14:29:42,465 [klippy_connection.py:_init_klippy_connection()] - GCode Output Subscribed
2022-12-06 14:29:45,474 [job_state.py:_handle_started()] - Job state initialized: standby
2022-12-06 14:29:45,749 [klippy_connection.py:_check_ready()] - Klippy ready
2022-12-06 14:29:46,233 [data_store.py:_init_sensors()] - Configuring available sensors: ['temperature_sensor raspberry_pi', 'temperature_sensor SKR_temp', 'temperature_sensor thermexp', 'temperature_sensor toolhead_MCU', 'temperature_sensor chamber', 'temperature_fan Stepper_Left', 'temperature_fan Stepper_Righ', 'temperature_fan electronics_bay', 'heater_bed', 'extruder']
2022-12-06 14:30:35,612 [websockets.py:write_to_socket()] - Websocket closed while writing: 2778639216
2022-12-06 14:30:40,419 [websockets.py:on_close()] - Websocket Closed: ID: 2778639216 Close Code: None, Close Reason: None, Pong Time Elapsed: 44.99
2022-12-06 14:31:26,415 [app.py:log_request()] - 101 GET /websocket (192.168.1.181) [_TRUSTED_USER_] 5.87ms
2022-12-06 14:31:26,416 [websockets.py:open()] - Websocket Opened: ID: 2818831824, Proxied: True, User Agent: Mozilla/5.0 (Linux; Android 12; SM-F711U1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Mobile Safari/537.36, Host Name: 192.168.1.90
2022-12-06 14:31:26,435 [websockets.py:_handle_identify()] - Websocket 2818831824 Client Identified - Name: mainsail, Version: 2.4.0, Type: web
2022-12-06 14:31:56,476 [websockets.py:write_to_socket()] - Websocket closed while writing: 2818831824
2022-12-06 14:32:01,421 [websockets.py:on_close()] - Websocket Closed: ID: 2818831824 Close Code: None, Close Reason: None, Pong Time Elapsed: 35.01
2022-12-06 14:33:41,789 [app.py:log_request()] - 101 GET /websocket (192.168.1.181) [_TRUSTED_USER_] 6.05ms
2022-12-06 14:33:41,790 [websockets.py:open()] - Websocket Opened: ID: 3050639280, Proxied: True, User Agent: Mozilla/5.0 (Linux; Android 12; SM-F711U1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Mobile Safari/537.36, Host Name: 192.168.1.90
2022-12-06 14:33:41,822 [websockets.py:_handle_identify()] - Websocket 3050639280 Client Identified - Name: mainsail, Version: 2.4.0, Type: web
2022-12-06 14:34:07,000 [websockets.py:on_close()] - Websocket Closed: ID: 3050639280 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 25.21
2022-12-06 14:34:07,348 [app.py:log_request()] - 101 GET /websocket (192.168.1.181) [_TRUSTED_USER_] 5.37ms
2022-12-06 14:34:07,349 [websockets.py:open()] - Websocket Opened: ID: 2819044816, Proxied: True, User Agent: Mozilla/5.0 (Linux; Android 12; SM-F711U1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Mobile Safari/537.36, Host Name: 192.168.1.90
2022-12-06 14:34:07,364 [websockets.py:_handle_identify()] - Websocket 2819044816 Client Identified - Name: mainsail, Version: 2.4.0, Type: web
2022-12-06 14:35:13,995 [websockets.py:build_error()] - JSON-RPC Request Error: 400
{'error': 'WebRequestError', 'message': "Malformed command 'PID_CALIBRATE HEATER=EXTRUDER 210'"}
Traceback (most recent call last):
  File "/home/pi/moonraker/moonraker/websockets.py", line 253, in execute_method
    result = await callback(params)
  File "/home/pi/moonraker/moonraker/websockets.py", line 351, in func
    user=sc.user_info))
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 481, in request
    return await self._request_standard(web_request)
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 536, in _request_standard
    return await base_request.wait()
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 608, in wait
    raise self.response
utils.ServerError: {'error': 'WebRequestError', 'message': "Malformed command 'PID_CALIBRATE HEATER=EXTRUDER 210'"}
2022-12-06 14:35:28,122 [websockets.py:build_error()] - JSON-RPC Request Error: 400
{'error': 'WebRequestError', 'message': "Unknown heater 'EXTRUDER'"}
Traceback (most recent call last):
  File "/home/pi/moonraker/moonraker/websockets.py", line 253, in execute_method
    result = await callback(params)
  File "/home/pi/moonraker/moonraker/websockets.py", line 351, in func
    user=sc.user_info))
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 481, in request
    return await self._request_standard(web_request)
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 536, in _request_standard
    return await base_request.wait()
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 608, in wait
    raise self.response
utils.ServerError: {'error': 'WebRequestError', 'message': "Unknown heater 'EXTRUDER'"}
2022-12-06 14:35:43,117 [websockets.py:build_error()] - JSON-RPC Request Error: 400
{'error': 'WebRequestError', 'message': "Malformed command 'PID_CALIBRATE HEATER=extruder 210'"}
Traceback (most recent call last):
  File "/home/pi/moonraker/moonraker/websockets.py", line 253, in execute_method
    result = await callback(params)
  File "/home/pi/moonraker/moonraker/websockets.py", line 351, in func
    user=sc.user_info))
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 481, in request
    return await self._request_standard(web_request)
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 536, in _request_standard
    return await base_request.wait()
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 608, in wait
    raise self.response
utils.ServerError: {'error': 'WebRequestError', 'message': "Malformed command 'PID_CALIBRATE HEATER=extruder 210'"}
2022-12-06 14:35:57,490 [websockets.py:build_error()] - JSON-RPC Request Error: 400
{'error': 'WebRequestError', 'message': "Malformed command 'PID_CALIBRATE HEATER=extruder target= 210'"}
Traceback (most recent call last):
  File "/home/pi/moonraker/moonraker/websockets.py", line 253, in execute_method
    result = await callback(params)
  File "/home/pi/moonraker/moonraker/websockets.py", line 351, in func
    user=sc.user_info))
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 481, in request
    return await self._request_standard(web_request)
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 536, in _request_standard
    return await base_request.wait()
  File "/home/pi/moonraker/moonraker/klippy_connection.py", line 608, in wait
    raise self.response
utils.ServerError: {'error': 'WebRequestError', 'message': "Malformed command 'PID_CALIBRATE HEATER=extruder target= 210'"}
2022-12-06 14:37:07,503 [websockets.py:write_to_socket()] - Websocket closed while writing: 2819044816
2022-12-06 14:37:12,353 [websockets.py:on_close()] - Websocket Closed: ID: 2819044816 Close Code: None, Close Reason: None, Pong Time Elapsed: 44.99
2022-12-06 14:37:19,247 [klippy_connection.py:wait()] - Request 'gcode/script' pending: 60.00 seconds
2022-12-06 14:38:19,249 [klippy_connection.py:wait()] - Request 'gcode/script' pending: 120.00 seconds
2022-12-06 14:39:19,250 [klippy_connection.py:wait()] - Request 'gcode/script' pending: 180.00 seconds
2022-12-06 14:41:43,648 [app.py:log_request()] - 101 GET /websocket (192.168.1.181) [_TRUSTED_USER_] 5.91ms
2022-12-06 14:41:43,649 [websockets.py:open()] - Websocket Opened: ID: 2789137552, Proxied: True, User Agent: Mozilla/5.0 (Linux; Android 12; SM-F711U1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Mobile Safari/537.36, Host Name: 192.168.1.90
2022-12-06 14:41:43,666 [websockets.py:_handle_identify()] - Websocket 2789137552 Client Identified - Name: mainsail, Version: 2.4.0, Type: web
2022-12-06 14:42:02,719 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2022-12-06 14:42:03,985 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2022-12-06 14:42:04,327 [klippy_connection.py:_init_klippy_connection()] - Webhooks Subscribed
2022-12-06 14:42:04,333 [klippy_connection.py:_init_klippy_connection()] - GCode Output Subscribed
2022-12-06 14:42:07,335 [job_state.py:_handle_started()] - Job state initialized: standby
2022-12-06 14:42:07,608 [klippy_connection.py:_check_ready()] - Klippy ready
2022-12-06 14:42:08,093 [data_store.py:_init_sensors()] - Configuring available sensors: ['temperature_sensor raspberry_pi', 'temperature_sensor SKR_temp', 'temperature_sensor thermexp', 'temperature_sensor toolhead_MCU', 'temperature_sensor chamber', 'temperature_fan Stepper_Left', 'temperature_fan Stepper_Righ', 'temperature_fan electronics_bay', 'heater_bed', 'extruder']
2022-12-06 14:43:00,157 [websockets.py:on_close()] - Websocket Closed: ID: 2789137552 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 6.49
2022-12-06 14:49:54,814 [app.py:log_request()] - 101 GET /websocket (192.168.1.181) [_TRUSTED_USER_] 5.39ms
2022-12-06 14:49:54,815 [websockets.py:open()] - Websocket Opened: ID: 3050638320, Proxied: True, User Agent: Mozilla/5.0 (Linux; Android 12; SM-F711U1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Mobile Safari/537.36, Host Name: 192.168.1.90
2022-12-06 14:49:54,830 [websockets.py:_handle_identify()] - Websocket 3050638320 Client Identified - Name: mainsail, Version: 2.4.0, Type: web
2022-12-06 14:50:08,805 [websockets.py:on_close()] - Websocket Closed: ID: 3050638320 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 3.98
2022-12-06 14:50:09,463 [klippy_connection.py:_process_status_update()] - Klippy has shutdown
2022-12-06 14:50:09,463 [power.py:_power_off_on_shutdown()] - Powering off device 'mainpower' due to klippy shutdown
2022-12-06 14:50:09,465 [proc_stats.py:_handle_shutdown()] - 
Moonraker System Usage Statistics:
System Time: 1670359779.730808, Usage: 1.59%, Memory: 29788 kB
System Time: 1670359780.733051, Usage: 1.58%, Memory: 29788 kB
System Time: 1670359781.734421, Usage: 5.5%, Memory: 29788 kB
System Time: 1670359782.734991, Usage: 1.59%, Memory: 29788 kB
System Time: 1670359783.735647, Usage: 1.59%, Memory: 29788 kB
System Time: 1670359784.736259, Usage: 1.54%, Memory: 29788 kB
System Time: 1670359785.737350, Usage: 1.57%, Memory: 29788 kB
System Time: 1670359786.737927, Usage: 1.54%, Memory: 29788 kB
System Time: 1670359787.738792, Usage: 1.58%, Memory: 29788 kB
System Time: 1670359788.739261, Usage: 1.61%, Memory: 29788 kB
System Time: 1670359789.740235, Usage: 1.62%, Memory: 29788 kB
System Time: 1670359790.742106, Usage: 1.6%, Memory: 29788 kB
System Time: 1670359791.745313, Usage: 5.34%, Memory: 29788 kB
System Time: 1670359792.744593, Usage: 1.72%, Memory: 29788 kB
System Time: 1670359793.744770, Usage: 1.62%, Memory: 29788 kB
System Time: 1670359794.746676, Usage: 1.46%, Memory: 29788 kB
System Time: 1670359795.743058, Usage: 51.19%, Memory: 29788 kB
System Time: 1670359796.748005, Usage: 1.36%, Memory: 29788 kB
System Time: 1670359797.750081, Usage: 2.1%, Memory: 29788 kB
System Time: 1670359798.751167, Usage: 2.08%, Memory: 29788 kB
System Time: 1670359799.751472, Usage: 2.12%, Memory: 29788 kB
System Time: 1670359800.753606, Usage: 2.51%, Memory: 29788 kB
System Time: 1670359801.756506, Usage: 5.77%, Memory: 29788 kB
System Time: 1670359802.756104, Usage: 2.2%, Memory: 29788 kB
System Time: 1670359803.757258, Usage: 2.15%, Memory: 29788 kB
System Time: 1670359804.762594, Usage: 2.12%, Memory: 29788 kB
System Time: 1670359805.755692, Usage: 2.71%, Memory: 29788 kB
System Time: 1670359806.755397, Usage: 1.12%, Memory: 29788 kB
System Time: 1670359807.758140, Usage: 1.17%, Memory: 29788 kB
System Time: 1670359808.756826, Usage: 1.35%, Memory: 29788 kB
CPU Temperature: 36.511
2022-12-06 14:50:09,481 [proc_stats.py:_handle_shutdown()] - Throttled Flags: 
2022-12-06 15:51:21,809 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.1.181
